docs/release-notes.rst

```diff
@@ -97,6 +123,14 @@ If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, t
 
 -------
 
+**Application services declaration delete of scaled config may result in an error 'HTTPError' object has no attribute 'message'**
+
+If an AS3 declaration with scale config is deleted using **L4-L7 App Services > View AS3 Declaration > Delete** button, the F5 ACI ServiceCenter may display an error  'HTTPError' object has no attribute 'message' instead of the actual error that BIG-IP responds with which is '503 Server Error'. 
```

Comment:
using the **L4-L7 App Services > View AS3 Declaration > Delete** button, 

the F5 ACI ServiceCenter may display an error stating the 'HTTPError' object has no attribute 'message'

that BIG-IP responds with, which is '503 Server Error'.